modules/openapi-generator/src/main/java/org/openapitools/codegen/config/CodegenConfigurator.java

```diff
@@ -497,7 +498,7 @@ public ClientOptInput toClientOptInput() {
         checkAndSetAdditionalProperty(gitRepoId, CodegenConstants.GIT_REPO_ID);
         checkAndSetAdditionalProperty(releaseNote, CodegenConstants.RELEASE_NOTE);
         checkAndSetAdditionalProperty(httpUserAgent, CodegenConstants.HTTP_USER_AGENT);
-
+        checkAndSetAdditionalProperty(virtualService,  CodegenConstants.VIRTUAL_SERVICE);
```

Comment:
I suggest not making it a top-level option as it's specified to the spring generator only